dlib/cmake_utils/find_blas.cmake

```diff
@@ -50,7 +58,19 @@ if (UNIX OR MINGW)
 
       # We need cblas since MATLAB doesn't provide cblas symbols.
       add_subdirectory(external/cblas)
-      set(blas_libraries  cblas  )
+      if("${CMAKE_SYSTEM}" MATCHES "Linux")
+        if("${LSB_RELEASE_ID_SHORT}" STREQUAL "Arch Linux" OR 
+	    "${LSB_RELEASE_ID_SHORT}" STREQUAL "Arch" OR 
+	    "${LSB_RELEASE_ID_SHORT}" STREQUAL "ArcoLinux" OR 
+	    "${LSB_RELEASE_ID_SHORT}" STREQUAL "EndeavourOS" OR 
+	    "${LSB_RELEASE_ID_SHORT}" STREQUAL "ManjaroLinux")
+	  	 set(blas_libraries "-lcblas;-lblas")
```

Comment:
Does `set(blas_libraries cblas blas)` not work?

Reply:
@davisking That is working too. Should change to `set(blas_libraries cblas blas)` ?

Reply:
So the problem is before only `cblas` with `blas` explicitly declared, they gone :laughing:

Reply:
Yes, `set(blas_libraries cblas blas)` is more appropriate.

Reply:
I have setted it to `set(blas_libraries cblas blas)`
Thank You Davis

Reply:
Thanks.  But this PR should still be changed to use `CHECK_FUNCTION_EXISTS` instead of running `lsb_release` and parsing the output.

Reply:
I see, I will check later for possible workaround, why `CHECK_FUNCTION_EXISTS` missed the blas library.

Reply:
`CHECK_FUNCTION_EXISTS` just isn't being called in the codepath you are using.

Reply:
Hi @davisking , I have tested latest commit on my machines, based on `CHECK_FUNCTION_EXISTS`. Thanks